website/integrations: Emby: cleanup (#23296)
goauthentik/authentik: website/integrations/media/emby/index.md

:::info Emby Premiere requirement
An [Emby Premiere](https://emby.media/premiere.html) subscription is required to use the official LDAP Authentication plugin.
:::

:::info
This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
:::

## authentik configuration

To support the integration of Emby with authentik, you need to create an LDAP application/provider pair in authentik, create a service account, and expose the provider through an LDAP outpost.

### Create an LDAP application, provider, and outpost in authentik

Follow the [LDAP provider setup](/docs/add-secure-apps/providers/ldap/create-ldap-provider/) to create the LDAP application, provider, service account, and outpost.

Complete the [service account creation](/docs/add-secure-apps/providers/ldap/create-ldap-provider/#create-a-service-account) and [LDAP search permission](/docs/add-secure-apps/providers/ldap/create-ldap-provider/#assign-the-ldap-search-permission-to-the-service-account) steps for the account Emby uses to connect to LDAP.

> When configuring the LDAP provider, set the following required configurations:

Reviewer: ```suggestion
When configuring the LDAP provider, set the following required settings:
```

- **Base DN**: `dc=company,dc=com`
- **Certificate**: select the certificate that Emby should trust for LDAPS.
- **TLS Server Name**: `ldap.company`

If access to the authentik LDAP application is restricted, allow the LDAP service account access via the application's [policy, group, or user bindings](/docs/add-secure-apps/applications/manage_apps/#use-bindings-to-control-access).

### Create an Emby access group

1. Log in to authentik as an administrator and open the authentik Admin interface.
2. Navigate to **Directory** > **Groups**.
3. Click **Create**, set **Name** to `emby_users`, and click **Create**.
4. Open the new group, click the **Users** tab, and click **Add existing user** to add the users who should have access to Emby.

## Emby configuration

> 1. Access your Emby server and log in using the administrator account or the currently configured local administrator credentials.

Reviewer: ```suggestion
1. Log in to Emby as an administrator.
```
